gen.py

```diff
@@ -211,23 +211,26 @@ def __init__(self, root):
         for el in self.root.iter("Function"):
             functions[el.get("name")] = Function(el, types, aliases)
 
-        function_names2 = self.get_function_names(2)
-        function_names3 = self.get_function_names(3)
+        function_names2_0 = self.get_function_names((2, 0))
+        function_names3_0 = self.get_function_names((3, 0))
+        function_names3_2 = self.get_function_names((3, 2))
 
         self.functions = []
-        for name in function_names3:
+        for name in function_names3_2:
             function = functions[name]
-            if name not in function_names2:
-                function.version = 3
-            else:
-                function.version = 2
+            if name not in function_names2_0:
+                function.version = (3, 0)
+            if name not in function_names3_0:
+                function.version = (3, 2)
```

Comment:
Can this part be rewritten without "not"?
```suggestion
            if name in function_names2_0:
                function.version = (2, 0)
            elif name in function_names3_0:
                function.version = (3, 0)
```

Reply:
sure